D_controllers.md

```diff
@@ -185,7 +204,7 @@ In order for the `render/3` function to work correctly, the controller must have
 
 There is also a rendering shortcut using the `plug/1` macro. Here's how it works.
 
-The first thing we need to do is to add a `plug :render` line to our controller.
+The first thing we need to do is to add a `plug :render` line to our controller in `web/controllers/page_controller.ex`.
```

Comment:
Changed. Still I think that `in` filename is more grammatically correct but not going to nitpit.